contracts/src/account/erc165.cairo

```diff
@@ -0,0 +1,45 @@
+// copied from https://github.com/OpenZeppelin/cairo-contracts/pull/583
```

Comment:
@cfal can you pin to the specific commit hash? the PR may change/update

Reply:
I attempted to get a commit for both this + account but the PR has since been changed. I think we can wait for it to get merged into master. (BTW account is actually from https://github.com/OpenZeppelin/cairo-contracts/pull/616 now)